### DISCUSSION TITLE
Bulk Checkout Accessories

### DISCUSSION THREAD

**Discussion by mmaxwell5** [snipe/snipe-it#11470] [Ideas]
IT would be great to be able to bulk checkout accessories like you are able to do with Assets.

**Comment by kamsobe** [comment]
This has been requested for years, and I just had my own feature request for this very thing merged into a long dead and closed feature request. As far as I can tell as an outsider, they seem to have no plans to implement this anytime soon.

Due to my own clients needs, I'll likely be starting over with a different asset tracking solution.

**Reply by Delirium-Sama** [reply]
we can already chekin chek out accessories ?  It's only consumable that we can't 

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/110677786/185973245-532da4b2-1e71-40c3-ae34-9255ee29f531.png">
